lib/Dialect/Substrait/IR/Substrait.cpp

```diff
@@ -397,6 +398,79 @@ void printFixedBinaryLiteral(AsmPrinter &printer, StringAttr value,
   printer << value;
 }
 
+static StringRef getTypeKeyword(Type type) {
+  return TypeSwitch<Type, StringRef>(type)
+      .Case<RelationType>([&](Type) { return "rel"; })
+      .Default([](Type) -> StringRef {
+        llvm_unreachable("unexpected 'llvm' type kind");
```

Comment:
is `llvm` intended here?

Reply:
Good catch! No, not intended. Will fix in the next iteration.